src/darwin/frida-helper-backend-glue.m

```diff
@@ -1556,26 +1567,284 @@ static gboolean frida_agent_context_init_functions (FridaAgentContext * self, Gu
 }
 
 static void
+frida_spawn_instance_create_dyld_data (FridaSpawnInstance * self)
+{
+  kern_return_t ret;
+  gboolean bret;
```

Comment:
Maybe `write_succeeded` is clearer. And changing the assertion to: `g_assert (write_succeeded);` so it's slightly easier to debug.